README.md

```diff
@@ -337,6 +337,8 @@ Started](https://docs.qmk.fm/#/?id=get-started) and refer to the [full Teensy
 compatibility chart](#reference-full-teensy-compatibility-chart) above to find
 the QMK branch to work with.
 
+TIP: When you flash your Teensy you'll be asked to press the reset button. The first time you flash, you'll need to press the push button on the Teensy itself. Default keyboard layouts map this ability to the Kinesis <kbd>progm</kbd> key. So for subsequent flashes, you can press the <kbd>progm</kbd> key instead.
```

Comment:
Alternately, we could change the key to <kbd>progm</kbd>+<kbd>F9</kbd> (since it is labelled `RESET` and used for that purpose with original board).

I proposed to do that here https://github.com/qmk/qmk_firmware/pull/18802#discussion_r1003600068, unfortunately, I have not gotten around to porting the changes to the default layout.